### ISSUE TITLE
autosklearn: fit, refit, fit_ensemble

### ISSUE THREAD

**Issue by zhuygln** [zhuygln/atoml#15]
when choosing cv, it was suggested to use fit and refit so that Refit all models found with fit to new data.
Necessary when using cross-validation. During training, auto-sklearn fits each model k times on the dataset, but does not keep any trained model and can therefore not be used to predict for new data points. This methods fits all models found during a call to fit on the data given. This method may also be used together with holdout to avoid only using 66% of the training data to fit the final model.

fit_ensemble(self, y, task=None, metric=None, precision='32', dataset_name=None, ensemble_nbest=None, ensemble_size=None)
Fit an ensemble to models trained during an optimization process.

All parameters are None by default. If no other value is given, the default values which were set in a call to fit() are used.

Calling this function is only necessary if ensemble_size==0, for example when executing auto-sklearn in parallel.
